test/typecheck.js

```diff
@@ -1174,3 +1174,189 @@ describe('negation', async () => {
     );
   });
 });
+
+describe('type system', () => {
+  async function assertTypeError(paramType, arg, message, extraBiblios = []) {
+    await assertLint(
+      positioned`
+        <emu-clause id="example" type="abstract operation">
+          <h1>
+            Example (
+              arg: ${paramType}
+            )
```

Comment:
`): ~unused~`, right?

Reply:
Isn't required and doesn't affect anything, but sure.